core/src/apps/monero/layout/address.py

```diff
@@ -0,0 +1,131 @@
+from micropython import const
+
+from trezor import loop, ui, wire
+from trezor.messages import ButtonRequestType
+from trezor.ui.button import ButtonDefault
+from trezor.ui.confirm import CONFIRMED, Confirm
+from trezor.ui.container import Container
+from trezor.ui.qr import Qr
+from trezor.ui.scroll import Paginated
+from trezor.ui.text import Text
+from trezor.wire import Context
+
+from apps.common import button_request
+from apps.common.confirm import confirm
+from apps.monero.layout import common
+
+if False:
+    from typing import Optional, Tuple
+
+if __debug__:
+    from apps.debug import swipe_signal
+
+
+async def show_address(
+    ctx, address: str, desc: str = "Confirm address", network: str = None
+):
+    pages = []
+    for lines in common.paginate_lines(common.split_address(address), 5):
+        text = Text(desc, ui.ICON_RECEIVE, ui.GREEN)
+        if network is not None:
+            text.normal("%s network" % network)
+        text.mono(*lines)
+        pages.append(text)
+
+    return await confirm(
+        ctx,
+        Paginated(pages),
+        code=ButtonRequestType.Address,
+        cancel="QR",
+        cancel_style=ButtonDefault,
+    )
+
+
+def render_scrollbar(pages: int, page: int) -> None:
+    BBOX = const(220)
+    SIZE = const(8)
+
+    padding = 14
+    if pages * padding > BBOX:
+        padding = BBOX // pages
+
+    X = const(232)
+    Y = (BBOX // 2) - (pages // 2) * padding
+
+    for i in range(0, pages):
+        if i == page:
+            fg = ui.FG
+        else:
+            fg = ui.GREY
+        ui.display.bar_radius(X, Y + i * padding, SIZE, SIZE, fg, ui.BG, 4)
```

Comment:
Only X was changed in this code compared to the `render_scrollbar` from `trezor/ui/scroll.py`. Do you prefer to a) extend the original function so X can be passed as a param (I assume no as it complicates API for just this use-case) or b) copy the code locally with minor modification?